cartoframes/io/carto.py

```diff
@@ -93,6 +96,11 @@ def to_carto(dataframe, table_name, credentials=None, if_exists='fail', geom_col
         ValueError: if the dataframe or table name provided are wrong or the if_exists param is not valid.
 
     """
+    def estimate_csv_size(gdf):
+        n = min(100, len(gdf))
```

Comment:
I would update 100 when we have the information of which value is enough for the precision we want for the estimation

Reply:
Yes, but <spoiler>, that looks like a good number